### PR TITLE
Create part_1_install_VIC

### PR DESCRIPTION
**Pull request by guillierf**

### CODE REVIEW

**Comment by guillierf** [comment]
test

File: tutorials/install/part_1_install_VIC [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    At least one Compute cluster with DRS enabled
+    Internet Access for VCH to download images from Docker Hub
+    Shared datastores to store VCH and containers (recommended minimum of 200Gb)
+    Allow communication on port 443 and 2377 between VCH and ESXi hosts
+    VCH needs to be reachable by Docker users for containers creation
+    VIC VM needs to be reachable by any admin user who needs to download the vic-machine utility
+    VIC VM needs to be reachable by Docker users for image upload and download (VIC VM hosts container registry)
+    VIC VM needs to be reachable by VCH for container instantiation using a local image
+    DVS switch with at least 2 Port Groups:
+        1 for for public communication (VCH to external world)
```
Comment (mhagen-vmware) [inline tutorials/install/part_1_install_VIC]: for for

File: tutorials/install/part_1_install_VIC [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        1 for for inter containers communication (recommended dedicated port group for each VCH)
```
Comment (mhagen-vmware) [inline tutorials/install/part_1_install_VIC]: for for

File: tutorials/install/part_1_install_VIC [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+VIC (vSphere Integrated Container) will be deployed as a VM from an OVA.
+
+VIC VM will host essentials services like the Harbor private registry and Admiral container management.
+
+VIC VM will host also the vic-machine utility that enables administrators to deploy VCH (Virtual Container Hosts).
+
+VCH will be instantiated as a VM under an automatically created resource pool in the Compute cluster.
+
+VCH behaves as a Docker endpoint where developer can target their system to in order to run Containers as VM (C-VM) on the ESXi cluster, up to the limit imposed by the resource pool.
```
Comment (mhagen-vmware) [inline tutorials/install/part_1_install_VIC]: *a* developer

Comment (mhagen-vmware) [reply tutorials/install/part_1_install_VIC]: system to -> docker commands

**Pull request closed without merging**